cdk.json

```diff
@@ -40,11 +40,11 @@
       "STACK_NAME_PREFIX": "dca",
       "ACM_CERT_ARN": "arn:aws:acm:us-east-1:631692904429:certificate/8644a975-2c1d-49ca-bc73-786745db0bc8",
       "VPC_CIDR": "10.255.70.0/24",
-      "STICKY":"FALSE",
-      "DESIRED_TASK_COUNT":"1"
+      "STICKY":"TRUE",
```

Comment:
changing this to true would be the exact same configs as `dev-sticky` context.  what's the point?

Reply:
`dev-sticky` and `dev-no-sticky` are temporary testing instances. I want `dev` to have the `dev-sticky` set up.